research/upgrade_interviews/index.md

```diff
@@ -0,0 +1,56 @@
+---
+parent: Research
+
+---
+# Upgrade Interviews
+
+*Date: July 2020*
+
+*Note: this is an article about a research activity on how users upgrade OpenShift. If you're looking for documentation about how to upgrade OpenShift, please go to [this link.](https://docs.openshift.com/container-platform/4.4/welcome/index.html)*
+
+## Background
+
+When considering major design changes to how users can upgrade their clusters in OpenShift 4, we wanted to be sure we understood when, why, and how our users upgrade. We interviewed 1 internal team and 5 customers. We had a mix of those running 4.x in production and still running 3.11 with 4.x in testing clusters. We used a semi-structured interview approach to allow room for us to explore important issues to our users we weren’t already aware of. Here are a few example questions from our interview guide.
+
+- How often have you upgraded in the past year or two?
+- Is this the same for different environments, e.g. dev and prod?
+- Do you want to upgrade more or less or the same?
+- What was the reason you last upgraded your cluster?
+- Is there any internal process that needs to happen before upgrading clusters?
+- How do you track the progress of an upgrade?
+
+## Findings
+
+### The upgrade ‘job map’
+Our interviews uncovered fundamental aspects of the upgrade process for a computing platform. There are some OpenShift 4 and 3 details, but these are good aspects to keep in mind when designing around the upgrade process for any platform. We borrowed the idea of a job map from the jobs-to-be-done framework to explain the process.
+
+![](img/upgrade_job_map.png)
+
+We found a few particular job map aspects worth writing out.
+
```

Comment:
I found these job map aspects very useful and what I would use as a designer to develop next steps. I suggest using bullets or something to make them stand out from commentary a little more.

Reply:
Good to know! I appreciate the designer-oriented point of view there, I'll add bullets